pkgs/data/misc/media-types/default.nix

```diff
@@ -0,0 +1,80 @@
+{ lib, stdenv, fetchFromGitLab, nixosTests }:
+
+stdenv.mkDerivation rec {
+  pname = "media-types";
+  version = "10.1.0";
+
+  src = fetchFromGitLab {
+    domain = "salsa.debian.org";
+    owner = "debian";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-vLQHjHrYShEBf4Py0vjpugbfY8HHnIIrmnxIDjHu5g8=";
+  };
+
+  outputs = [ "out" "doc" ];
+
+  installPhase = ''
+    install -D -m0444 mime.types $out/etc/mime.types
+    install -D -m0444 debian/bug-presubj $doc/share/doc/media-types/presubj
+    install -D -m0444 debian/changelog $doc/share/doc/media-types/changelog
+    install -D -m0444 debian/copyright $doc/share/doc/media-types/copyright
+
+    # Generate nginx mime-types
+    mkdir -p  $out/etc/nginx
+    echo "types {" > $out/etc/nginx/mime.types
+    cat mime.types | sed -e '/^#.*/d' -e '/^$/d' -e 's/.*/\ \ &;/' -ne '/[^[:space:]][[:space:]]\{1,\}[^[:space:]]/p' >> $out/etc/nginx/mime.types
+    echo "}" >> $out/etc/nginx/mime.types
+
+    # Remove duplicate mime-types extensions
+    substituteInPlace $out/etc/nginx/mime.types \
+      --replace "application/x-maker				frm maker frame fm fb book fbdoc;" \
+                "application/x-maker				maker frame fb book fbdoc;" \
+      --replace "application/x-qgis				qgs shp shx;" \
+                "application/x-qgis				qgs;" \
+      --replace "application/x-scilab				sci sce;" \
+                "application/x-scilab				sci;" \
+      --replace "audio/AMR					amr AMR;" \
+                "audio/AMR					amr;" \
+      --replace "audio/AMR-WB					awb AWB;" \
+                "audio/AMR-WB					awb;" \
+      --replace "audio/EVRC-QCP					qcp QCP;" \
+                "audio/EVRC-QCP				qcp;" \
+      --replace "chemical/x-mdl-sdfile				sd sdf;" \
+                "chemical/x-mdl-sdfile			sd;" \
+      --replace "chemical/x-mopac-input				mop mopcrt mpc zmt;" \
+                "chemical/x-mopac-input			mop mopcrt zmt;" \
+      --replace "chemical/x-ncbi-asn1-binary			val aso;" \
+                "chemical/x-ncbi-asn1-binary			val;"\
+      --replace "image/j2c					j2c J2C j2k J2K;" \
+                "image/j2c					j2c j2k;" \
+      --replace "image/vnd.globalgraphics.pgb			PGB pgb;" \
+                "image/vnd.globalgraphics.pgb			pgb;" \
+      --replace "text/x-tcl					tcl tk;" \
+                "text/x-tcl					tk;"
+
+    sed -i $out/etc/nginx/mime.types \
+      -e '/application\/mac-compactpro\t.*/d' \
+      -e '/audio\/x-gsm\t.*/d' \
+      -e '/chemical\/x-chemdraw\t.*/d' \
+      -e '/chemical\/x-cif\t.*/d' \
+      -e '/chemical\/x-cml\t.*/d' \
+      -e '/chemical\/x-ncbi-asn1-spec\t.*/d' \
+      -e '/chemical\/x-pdb\t.*/d' \
+      -e '/image\/x-jg\t.*/d' \
+      -e '/text\/x-csh\t.*/d' \
+      -e '/text\/x-sh\t.*/d'
```

Comment:
...or this one. Even with the test because ofborg is not displaying them as hard failures in red.